.github/workflows/dotnet-format.yml

```diff
@@ -13,7 +13,7 @@ jobs:
       - name: Setup .NET
         uses: actions/setup-dotnet@v4
         with:
-          dotnet-version: 8.0.x
+          dotnet-version: 9.0.x
```

Comment:
Looks good to me, just want to confirm though that given we're now using the .NET 9 SDK that we're still releasing a .NET 8 library?

Reply:
The target framework remains net8.0 in the csproj `TargetFrameworks>net8.0;</TargetFrameworks>`, so my understanding is that it all still builds as .net 8 library, even if we use SDK 9 to build it.